python/lsst/daf/base/exampleLib.py

```diff
@@ -0,0 +1,12 @@
+from _exampleLib import *
```

Comment:
Please don't use from X import * in modules (other than in `__init__.py` files to hoist symbols into the package namespace). It makes it impossible to tell where symbols came from, reducing the ability of linters to work.